ludwig/combiners/combiners.py

```diff
@@ -514,28 +570,63 @@ def output_shape(self) -> torch.Size:
         return self.tabnet.output_shape
 
 
+# TODO(ksbrar): Refactor transformers into using base class for common attrs?
 @dataclass
 class TransformerCombinerConfig:
     num_layers: int = schema.PositiveInteger(default=1)
+    """TODO (default: 1)."""
+
     hidden_size: int = schema.NonNegativeInteger(default=256)
+    """The size of the hidden representation within the transformer block. It is usually the same as `embedding_size`,
```

Comment:
Does the point about `embedding_size` still make sense? This will be sent out in the schema, so does it make sense to keep this in the docstring?

Reply:
Maybe: `The number of hidden units of the TransformerStack as well as the dimension that each incoming input feature is projected to before feeding to the TransformerStack.`